relay-server/src/actors/project.rs

```diff
@@ -725,46 +732,24 @@ impl Project {
         })
     }
 
-    fn check_envelope_scoped(&mut self, message: CheckEnvelope) -> CheckEnvelopeResponse {
+    pub fn check_envelope_scoped(&mut self, message: CheckEnvelope) -> CheckEnvelopeResponse {
```

Comment:
_nit_: I'm realizing that the naming of the two functions `check_envelope_scoped` and `check_envelope` would be more clear the other way around. Mind updating that while we're at it?

Reply:
sure